bodhi/tests/server/scripts/test_approve_testing.py

```diff
@@ -333,3 +333,41 @@ def test_non_autokarma_update_without_mandatory_days_in_testing(self):
             c.user.name
             for c in self.db.query(models.Comment).order_by(models.Comment.timestamp).all()]
         self.assertEqual(usernames, [u'guest', u'anonymous', u'hunter2'])
+
+    @patch.dict(config, [('fedora.mandatory_days_in_testing', 14)])
+    def test_subsequent_comments_after_initial_push_comment(self):
+        """
+        If a user edits an update after Bodhi comments saying that it can be
+        pushed, Bodhi should comment again saying that it has been in testing
+        long enough to be pushed to stable.
+        """
+        update = self.db.query(models.Update).all()[0]
+        update.request = None
+        update.status = models.UpdateStatus.testing
+        update.date_testing = datetime.now() - timedelta(days=14)
+        text = unicode(config.get('testing_approval_msg') % update.days_in_testing)
+        update.comment(self.db, text, author=u'bodhi')
+
+        with patch(
+                'bodhi.server.scripts.approve_testing._get_db_session', return_value=self.db):
+            approve_testing.main(['nosetests', 'some_config.ini'])
```

Comment:
You probably don't need this line, since line 349 already adds the comment. So you could drop this line, or you could drop lines 348 and 349.